cfme/tests/configure/test_paginator.py

```diff
@@ -151,6 +159,15 @@ def test_paginator_details_page(appliance, place_info, schedule):
             table[0].click()
     assert check_paginator_for_page(view) == paginator_expected_result
 
+    if check_paginator_for_page(view):
+        paginator = view.paginator
+        items_selector = Dropdown(view, '{} Items'.format(paginator.items_per_page))
+        msg = 'Not all options are present in items per page'
+        if view.extra.appliance.version < '5.11':
+            assert set(items_selection_5_10).issubset(items_selector.items), msg
```

Comment:
shouldn't those be equal ?

Reply:
@izapolsk good point. I agree, we don't want to see any other weird options introduced in this dropdown